docs/src/main/asciidoc/spring-cloud-aws.adoc

```diff
@@ -121,21 +125,21 @@ logging.level.com.amazonaws.internal.InstanceMetadataServiceResourceFetcher=erro
 ====
 
 ==== SDK credentials configuration
-In order to make calls to the Amazon Web Service the credentials must be configured for the the Amazon SDK. Spring Cloud AWS
-provides support to configure an application context specific credentials that are used for _each_ service call for requests done
-by Spring Cloud AWS components, with the exception of the Parameter Store and Secrets Manager Configuration.
+In order to make calls to Amazon web services, credentials must be configured for the Amazon SDK. Spring Cloud AWS
```

Comment:
It should stay "the Amazon Web Services" as this is what AWS stands for..?

Reply:
Agreed. Addressed. Thanks Maciej.